packages/browser/src/tests/browser.test.ts

```diff
@@ -46,6 +46,26 @@ describe('Browser', () => {
     });
   });
 
+  describe('isShopifyPOSApp', () => {
+    it('returns true for Shopify POS', () => {
+      const userAgent =
+        'com.jadedpixel.pos Shopify POS/5.11.0 (iPad; iOS 9.3; Scale/2.00)';
```

Comment:
I'm not seeing `com.jadedpixel.pos` suffix on the user-agent coming from the POS app:

<img width="300" alt="Screenshot 2024-02-13 at 3 15 28 PM" src="https://github.com/Shopify/quilt/assets/1030977/659b3ab1-1fbb-4901-844b-cc5d8914623e">


It appears that this suffix was included for first-party devices as part of the workaround found [here](https://github.com/Shopify/pos-next-react-native/pull/14837/files). Perhaps we could remove it from the tests, or even better, introduce an additional test to cover this case?